app/src/org/odk/collect/android/views/ODKView.java

```diff
@@ -434,10 +426,6 @@ public void removeWidget(View v){
      * @return Index of question's view in mView.
      */
     private int getViewIndex(int questionIndex) {
-        // Account for progress bar
-        if (mProgressEnabled) {
```

Comment:
yeah, `mProgressEnabled` was never set, so it was always false, so that conditional was never satisfied.